nle/nethack/actions.py

```diff
@@ -211,6 +212,7 @@ class Command(enum.IntEnum):
 for action in NON_RL_ACTIONS + tuple(TextCharacters):
     _USEFUL_ACTIONS.remove(action)
 _USEFUL_ACTIONS.append(TextCharacters.SPACE)
+_USEFUL_ACTIONS.append(TextCharacters.DOLLAR)
```

Comment:
This line is unnecessary since `NetHackChallenge-v0` uses `nethack.ACTIONS` not `nethack.USEFUL_ACTIONS`.   but we can still add here.